app.py

```diff
@@ -46,29 +46,31 @@ class Subjects(db.Model):
 	subject = db.Column(db.Text, unique=True, nullable=False)
 
 
-class Groups(db.Model):
-	__tablename__ = 'groups'
+class Channels(db.Model):
+	__tablename__ = 'channels'
 	id = db.Column(db.Integer, primary_key=True)
-	group = db.Column(db.Text, unique=True, nullable=False)
+	channel = db.Column(db.Text, unique=True, nullable=False)
 
 
 class Posts(db.Model):
 	__tablename__ = 'posts'
 	id = db.Column(db.Integer, primary_key=True)
-	user = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+	user = db.relationship("Users", backref="users")
```

Comment:
great, put it last(ufter time_updated) because uts not a column

Reply:
done 👌🏻